config/rbac/networkplugin_syncer_cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: submariner-networkplugin-syncer
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - secrets
+      - nodes
+    verbs:
+      - get
+      - list
```

Comment:
Can we remove - list for "secrets and configmaps", for nodes we need it.

We will limit a little bit the security this way.. at least the name of the secret/configmap needs to be know to be accessible.